README.md

```diff
@@ -1,19 +1,112 @@
-## Node Curriculum Assignment 
+# Node Dependency Management
+You've already learned how to build a small, single file application in Node.js - congrats!
 
+Now we'll discuss how to break up your app into smaller related chunks of code called modules to help keep things organized as your code base grows.
 
-## What the Student Already Knows
+## Why modules?
+Think of your application as though it were a business. In the early stages when your team is small, having everyone work in the same room is totally reasonable.
 
-Students have covered the basic structure and syntax of small (single-file) command-line applications. They have not, up until this point, used `require`
+As your company scales and you hire employees to perform specific functions, you'll eventually need to organize your company into departments; each with its own special purpose (eg: sales, engineering, project management).
 
-## Learning Goal 
+Modules provide the same organizational structure and value for your application!
 
-This lesson introduces students to dependency management in Node.js . It should demonstrate the different ways that `require()` works — highlighting some of its quirks and virtues — and it should also introduce them to `npm`
+## Dependencies and NPM
+We'll introduce you to the concept of modules (which we'll sometimes refer to as packages) and dependencies by creating a basic app with Express - the popular Node web application framework.
 
-## Learning Objectives- Students should be able to: 
+Express can be installed into your application via NPM, Node's package (module) management system. We can use the NPM command line tool that ships with Node to install packages into our application from the NPM registry.
```

Comment:
This might be a little dense for beginning students, but I think you cover it sufficiently below -- I actually really like that this line gives students a preview of what's come up in the lesson.